### PR TITLE
fix: make coins device-agnostic when bootloader mode

### LINKED ISSUE #4810: Not possible to change Coins settings when model T in bootloader mode
**Describe the bug**
it's possible to change Coins settings when model T in bootloader mode.
Model One works as expected but when model T connected it reports `Update required
`

**Steps to reproduce:**
1. Start latest dev or production version of Suite
2. Connect Trezor model T
3. Observe Coins tab in Settings

**Info:**
 - Suite version: desktop 22.1.1 (e7262e597f6af0487e17ef055a2583425c7a5302)
 - Browser: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) TrezorSuite/22.1.1 Chrome/94.0.4606.81 Electron/15.3.0 Safari/537.36
 - OS: MacIntel
 - Screen: 1680x1050
 - Device: model T 2.4.3 regular

**Screenshots:**

_Trezor model T in bootloader mode_ 

obrazek

_Trezor model One in bootloader mode_ 

obrazek

### PR DESCRIPTION
**Pull request by karliatto**
Closes https://github.com/trezor/trezor-suite/issues/4810

### CODE REVIEW

File: packages/suite/src/components/suite/CoinsGroup/CoinsList.tsx [hunk 1]
```diff
@@ -62,11 +63,14 @@ const CoinsList = ({
...
+                const isUnavailable = !!unavailable && !isBootloaderMode;
                 // Coin is not available because:
                 // - connect reports this in device.unavailableCapabilities (not supported by fw, not supported by connect)
                 // - suite considers device 'locked'
                 // - suite does not support it which is defined in network.ts
-                const disabled = !!unavailable || locked || !supportedBySuite;
+                const disabled = isUnavailable || locked || !supportedBySuite;
 
                 return (
                     <Tooltip
```
Comment (matejkriz) [inline packages/suite/src/components/suite/CoinsGroup/CoinsList.tsx:78]: tooltip needs to be handled too (Trezor T, latest FW shows "Update required" for all coins when in bootloader mode)
image

Comment (karliatto) [reply packages/suite/src/components/suite/CoinsGroup/CoinsList.tsx:78]: ohh, good catch! I will handle that!

Comment (karliatto) [reply packages/suite/src/components/suite/CoinsGroup/CoinsList.tsx:78]: @matejkriz it should be good now

**Review by matejkriz — APPROVED** [review]

**Comment by matejkriz** [comment]
@karliatto please squash and feel free to merge 👍

**Pull request merged**